packages/evolution-backend/src/services/auth/participantAuthModel.ts

```diff
@@ -44,6 +46,11 @@ class ParticipantAuthModel extends AuthModelBase<ParticipantModel> {
         });
         const user = this.newUser(userAttribs);
 
+        // Send a welcome email to the new user if the server is configured to do so
+        if (config.auth?.welcomeEmail) {
```

Comment:
You should put the `===true` if that is the only value you want to put, or use __booleish. Here, the string "false" would return true

Reply:
C'est bon, j'ai fait la correction. On peut merge. :)